documentation/solution-architecture.md

```diff
@@ -0,0 +1,151 @@
+
+- clerk-frontend
+- authorisation-service
+- client-frontend
+- adminstrator-frontend
+
+Validation does not require usage of this server; validators run their own embedded local ephemeral service (without client-server communication) just to hold information temporarily and use it for their own validation process executed within a client side validation app. As NLX certificates in the end are to originate from official root certificates (conform EiDAS/EIF), validation for now just involves standard techniques around X.509 certificates and can even be done offline.
+
+Below is a more detailed component view:
+
+![](images/components.png)
+
+## Infrastructural layer
+
+![](images/deployment.png)
+
+The solution is hosted on server(s) in control of the issuer and used by the devices (broad range of desktop computers, mobile phones or tablets) in control of citizens, clerks, validators and administrators. The server at the issuer is accessible on the intranet of the issuer and publicly through an existing web server (in a controlled way through a reverse proxy) that also will serve the client side web application.
+
+To enable easy integration, the waardepapieren service along with a nlx outway are available as docker containers as well as examples of integrations of the clerk and client front end applications. All these can be started using docker compose. The service and front end applications are configurable from a single configuration file. The amount of code to create to integrate the solution in to website / backoffice
+is made as minimal as possible.
+
+## Key decisions
+
+- scope : the waardepapieren service will at least provide in a single product that can be issued.
+- scope : the information being signed is that of the result data of a NLX call given the BSN number as argument though this argument is configurable
+- scope : only the paper wallet is in scope as supported verifiable credential solution
+- scope : identification and authentication is performed either by the clerk-frontend user in the local intranet the waardepapieren service runs in, or through the authentication service integrated within the website through which the service is made accessible. The validation app autenticates a proof through well established techniques around X.509 certificates issued conform EiDAS/EIF.
+- scope : the topic of alignment with W3C Verifiable Credential standard, ETSI TS 103 171 and REGULATION (EU) No 910/2014 is deferred to ongoing development of Discipl
+
+## Future directions
+
+The solution is prepared to be used in different kind of situations and set ups and use cases in which other types of platforms can be used instead of ephemeral, NLX or paper wallet. The idea is that it is prepared for usage in automated facilitation in requirement resolving in which for instance when a client needs something that requires a signed proof of some issuer, the proof can be automatically and conveniently requested as need
```

Comment:
situations and set ups and use cases -> use cases (situations/set ups just makes this sentences more difficult to read)

Maybe split the second sentence in two or three sentences.

Reply:
ok